tests/sos_tests.py

```diff
@@ -389,7 +389,7 @@ def manifest(self):
                 self._manifest = json.loads(content)
             except Exception:
                 self._manifest = ''
-                self.warning('Could not load manifest for test')
+                self.log.debug('Could not load manifest for test')
```

Comment:
Why warning verbosity replaced by debug, please?

Reply:
It's to fix:
```
tests/sos_tests.py:392:16: E1101: Instance of 'BaseSoSReportTest' has no 'warning' member (no-member)
```

Reply:
I believe that should be `warn` in that case, instead of `warning`.

Reply:
Doesn't work: https://github.com/sosreport/sos/pull/3747/checks?check_run_id=28716881665

Reply:
I don't see any `warn` or `warning` method anywhere in the class chain. I am also not sure where this message is *supposed* to go. So if `self.log.debug` isn't good, maybe just `print` is ok?

Reply:
Try `self.log.warn`

Reply:
`self.log.warn` works.

Reply:
basically, is coming from the Test class from avocado. If you look through that class; you'll see the following line

```
self.log.warn = self.log.warning = record_and_warn
```

So, you could have used `self.log.warn` or `self.log.warning`

Reply:
Thanks, that makes sense. Should be good now then.